nodejs/aws-serverless/function.ts

```diff
@@ -36,11 +36,16 @@ const defaultComputePolicies = [
 ];
 
 export function createLambdaFunction<E, R>(
-    name: string, handler: Handler<E, R>, opts?: ResourceOptions): aws.lambda.Function {
+    name: string, handler: Handler<E, R>, opts?: ResourceOptions, additionalPolicies?: string[]): aws.lambda.Function {
```

Comment:
Since we export this function, we probably want to think about how best to expose this and other options.  Should we accept an `aws.serverless.FunctionOptions` here?  With the intention that that will eventually become something that is effectively `aws.lambda.FunctionArgs`?

Reply:
yes.  we really should do that.  would you prefer that in this PR, or a followup?